tests/testMethods.py

```diff
@@ -156,6 +156,58 @@ def testAssertCoordsNearlyEqual(self):
             coord2 = afwCoord.IcrsCoord(raDecDeg[0] + 360*afwGeom.degrees, raDecDeg[1])
             self.assertCoordsNearlyEqual(coord0, coord2, maxDiff=1e-7*afwGeom.arcseconds)
 
+    def testAssertSpherePointsAlmostEqual(self):
```

Comment:
overall pleas consider not having magic numbers in function calls, separate them out to variables so they can be reused